app/controllers/api/v2/moves_filtered_actions.rb

```diff
@@ -0,0 +1,5 @@
+module Api::V2
```

Comment:
I had to add this module as the `Api::V2::MovesActions` was not being extended even when I explicitly added the following line to the filtered controller itself: `extended Api::V2::MovesActions`